src/DevBetterWeb.Web/Interfaces/ILeaderboardService.cs

```diff
@@ -10,4 +11,5 @@ namespace DevBetterWeb.Web.Interfaces;
 public interface ILeaderboardService
 {
 	Task<List<BookCategoryDto>> SetBookCategoriesAsync();
+	Task<BookDetailsViewModel?> GetBookDetailsAsync(string bookId);
```

Comment:
Why is this on a leaderboard service? The list of members who have read a book isn't a leaderboard, it it?

Reply:
@ardalis 
So is better to make a new service for book details or just add this code to the book details page
```csharp
	private async Task<BookDetailsViewModel?> GetBookDetailsAsync(string bookId)
	{
		var bookDetails = await _bookService.GetBookByIdAsync(int.Parse(bookId));
		if (bookDetails == null)
		{
			return null;
		}
		bookDetails = await _filteredLeaderboardService.RemoveNonCurrentMembersFromBookDetailsAsync(bookDetails);

		return bookDetails;
	}
```

Reply:
done